src/apps/music-player/musicPlayer.css

```
@@ -0,0 +1,616 @@
+}
+
+
+/* ================================
+   RESPONSIVE
+================================ */
+
+@media (max-width: 700px) {
+
+    .rage-music-player .music-playlist {
+
+        width: 35%;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)